ls1intum/Artemis
# `General`: Improve favicon logo for light mode

### Checklist
#### General

### Motivation and Context
This PR tries to resolve the following Git Hub Issue #3658. 
Original Logo PR : https://github.com/ls1intum/Artemis/pull/3626

The issue was that the logo was not clearly visible in the tab for people with bright/light tabs (light-mode)
![image](https://user-images.githubusercontent.com/65814168/143428909-0d47812e-d901-4f10-b1fc-28dca69bcfc0.png)

I discussed this isse with @krusche once more and we decided to do the following:
Based on the original logo PR and the final survey we simply pick the second place for some favicons
![image](https://user-images.githubusercontent.com/65814168/143429106-ebc7dc8e-7cdf-4ea2-8411-1a8c4feb12d6.png)
![image](https://user-images.githubusercontent.com/65814168/143429151-eed3626f-5c92-4193-a3a5-d6327147ccc4.png)

I did not change the main logo.png it should stay the same!

### Description
I additionally moved the icons/logos to a subfolder to not make the main webapp folder less chaotic. 
![image](https://user-images.githubusercontent.com/65814168/143439891-c38d51f1-316b-4e4d-ba0e-d3edb1f6db9f.png)


This is how it looks for white-mode users (e.g. Edge)
![image](https://user-images.githubusercontent.com/65814168/143430839-17095e02-ddae-4d70-a68d-2157bfe07f0e.png)

I am aware that this is not a perfect solution. A perfect one would be to differentiate/toggle dynamically between light and dark mode. But we do not have such a feature as of now.
E.g possible issues for dark mode users could appear now:
Incognito mode for Edge
![image](https://user-images.githubusercontent.com/65814168/143430962-6bb39bb0-e2ab-4fcf-9731-be12f4c419e8.png)

I myself use dark mode so yeah:
![image](https://user-images.githubusercontent.com/65814168/143431025-0f3c98ac-06f8-43b3-862e-e20a9738e185.png)
![image](https://user-images.githubusercontent.com/65814168/143431035-b0ec5480-f340-4f69-8e82-e1d12e102c4a.png)
(Especially because I have a custom chrome-browser skin that is blue-ish. Such edge-cases can not even be caught by light/dark mode I fear.
![image](https://user-images.githubusercontent.com/65814168/143431234-a39f1014-afb1-4555-a228-54b65ebbf45c.png)

But I think this is still a good chance because the majority will use white mode.

### Steps for Testing

1. Log in to Artemis and check the main logo and the favicon in the tab
2. Try to use multiple different browsers/devices to check if everything is fine

### Review Progress
#### Code Review
#### Manual Tests

### Screenshots
Here you can see the design choices/changes I took for the favicon/logo.
I used the following page: https://realfavicongenerator.net/
![image](https://user-images.githubusercontent.com/65814168/143426117-6a13a925-a67f-4c0f-8ae8-d8f8a2737705.png)
![image](https://user-images.githubusercontent.com/65814168/143426180-6ae736c4-ab23-426f-9b08-12d0f8f1cf3d.png)
![image](https://user-images.githubusercontent.com/65814168/143426232-f671ae4b-543f-406a-9e79-81a763316917.png)
![image](https://user-images.githubusercontent.com/65814168/143426261-f2fd103a-dd7c-4524-9eb6-a3482f9f9a24.png)
![image](https://user-images.githubusercontent.com/65814168/143426307-c0eef64a-2199-4cf3-818f-d99e9c0465f2.png)


No test should be influenced by this PR I only changed a couple of images.
The failing tests are unrelated to this PR (locally the tests work)
![image](https://user-images.githubusercontent.com/65814168/143434901-8fa9a7a4-1a98-427e-bc0d-fab6eab69e67.png)
